modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/image_gallery_display/view.jsp

```diff
@@ -90,7 +90,7 @@ String[] mediaGalleryMimeTypes = dlPortletInstanceSettings.getMimeTypes();
 	contextObjects="<%= contextObjects %>"
 	displayStyle="<%= displayStyle %>"
 	displayStyleGroupId="<%= displayStyleGroupId %>"
-	entries="<%= DLAppServiceUtil.getGroupFileEntries(scopeGroupId, 0, folderId, mediaGalleryMimeTypes, status, 0, SearchContainer.MAX_DELTA, null) %>"
+	entries="<%= DLAppServiceUtil.getGroupFileEntries(repositoryId, 0, folderId, mediaGalleryMimeTypes, status, 0, SearchContainer.MAX_DELTA, null) %>"
```

Comment:
I think we need to call `getFileEntries` here; we're no longer using the group, but the repositoryId (and these are not always equivalent).

Reply:
On the DLAppServiceImpl we use it the same as a repository

```
	public List<FileEntry> getGroupFileEntries(
			long groupId, long userId, long rootFolderId, String[] mimeTypes,
			int status, int start, int end,
			OrderByComparator<FileEntry> orderByComparator)
		throws PortalException {

		Repository repository = getRepository(groupId);

		return repository.getRepositoryFileEntries(
			userId, rootFolderId, mimeTypes, status, start, end,
			orderByComparator);
	}
```
vs 
```

	public List<FileEntry> getGroupFileEntries(
			long groupId, long userId, long rootFolderId, String[] mimeTypes,
			int status, int start, int end,
			OrderByComparator<FileEntry> orderByComparator)
		throws PortalException {

		Repository repository = getRepository(groupId);

		return repository.getRepositoryFileEntries(
			userId, rootFolderId, mimeTypes, status, start, end,
			orderByComparator);
	}
```
And we would lost the status variable (or made another method with the status)

Reply:
But they are not the same. One calls the getRepositoryFileEntries, the other one getFileEntries. The former returns a plain list of all file entries; the latter only the immediate children of the given folderId.

Anyway, lets keep the current behavior (getGroup* call with repositoryId), and we'll check later with Pablo to see if we can remove that in the future.

Reply:
This is sooo confusing